packages/infrastructure/main.tf

```diff
@@ -0,0 +1,178 @@
+resource "azurerm_resource_group" "this" {
+  name     = "rg-${var.name}-${var.env}-${var.location}"
+  location = var.location
+}
+
+resource "azurerm_cosmosdb_account" "slack_db" {
```

Comment:
Is there any way to split this file up and keep all `slack bot`-related resources in it's own file called something like `slack.tf`. That way when reading terraform code it's easier to understand whats common resources and whats for only one service.

Reply:
Yes. But to keep within terraform best practices this is best done with modules. This in my opinion is future work